SocialFilter/sections/specific.txt

```diff
@@ -78,6 +78,7 @@ ansa.it##.bt-news-flip
 ansa.it##.social-menu li.button-wrap
 slovoidilo.ua##.wrap_addthis_inline_bottom
 bykvu.com##.article-share-box
+abplive.com##.mt-4 > a[href^="https://www.youtube.com/"]
```

Comment:
I think this rule is better:
`abplive.com##amp-img[src="https://cdn.abplive.com/images/uncut-band-amp-article.png"]`

Reply:
or `a[href*="?sub_confirmation="]`